ares/n64/pif/pif.cpp

```diff
@@ -32,9 +32,16 @@ auto PIF::main() -> void {
 auto PIF::power(bool reset) -> void {
   Thread::reset();
 
-  string pifrom = Region::PAL() ? "pif.pal.rom" : "pif.ntsc.rom";
-  if(auto fp = system.pak->read(pifrom)) {
-    rom.load(fp);
+  if(Model::Aleck64()) {
+    string pifrom = "pif.rom";
+    if (auto fp = cartridge.pak->read(pifrom)) {
+      rom.load(fp);
+    }
```

Comment:
Is this going to be committed to mia/Firmware?
Can we maybe name it `pif.aleck64.rom` or something more explicit?

Reply:
Done